calabash-cucumber/lib/calabash-cucumber/usage_tracker.rb

```diff
@@ -131,26 +147,41 @@ def used_cucumber?
       #
       # Collect a hash of usage info.
       def info
-        {
+
+        allowed = allowed_to_track
+
+        return {} if allowed == "none"
+
+        # Events only
+        hash = {
           :event_name => "session",
```

Comment:
Perhaps I missed it, but aren't we also tracking some events besides `session`? I haven't seen other event types listed in the previous PRs, or you have just not gotten there?

I imagine that each calabash command line command might be considered an event. E.g.,
- `calabash-ios setup` #defunkt as it may be
- `calabash-ios gen`
- `calabash-ios console`
- `cucumber`

To which does "`session`" refer?